### PR TITLE
Find similar incidents fix - support list in context

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

File: Scripts/FindSimilarIncidents/FindSimilarIncidents.py [hunk 1]
```diff
@@ -85,8 +85,6 @@ def nested_dict_flatted(d, parent_key='', sep='.'):
         items = []  # type: ignore
         for k, v in d.items():
             new_key = parent_key + sep + k if parent_key else k
-            if isinstance(v, list) and len(v) > 0:
```
Comment (anara123) [inline Scripts/FindSimilarIncidents/FindSimilarIncidents.py]: why do you remove this?

**Comment by anara123** [comment]
is there related issue?

**Comment by erezh31** [comment]
No issue, came from a customer

File: Scripts/FindSimilarIncidents/FindSimilarIncidents.py (hunk 1 above)
Comment (erezh31) [reply Scripts/FindSimilarIncidents/FindSimilarIncidents.py]: to support list comparison - now it's consider only the fist element if it's list - and it's not a good enough logic for some use-cases

**Pull request merged**